infrastructure/src/main/java/org/corfudb/infrastructure/SequencerServer.java

```diff
@@ -178,6 +169,7 @@ public SequencerServer(ServerContext serverContext) {
         final long expireDuration = 20;
 
         conflictToGlobalTailCache = Caffeine.newBuilder()
+                .maximumSize(10000)
```

Comment:
Yes, this PR is not ready yet. I'm still in the process of setting a reasonable size.